service-commands/scripts/provision-dev-env.sh

```diff
@@ -46,6 +46,23 @@ function setup_ssh_timeouts() {
     sudo /etc/init.d/ssh restart
 }
 
+function setup_system_limits() {
+    # nofile - max number of open file descriptors https://stackoverflow.com/a/1213069
+    # nproc - max number of processes
+    # as - address space limit (KB)
+    echo "root - nofile $(cat /proc/sys/fs/nr_open)" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    echo "root - nproc 96118" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    echo "root - as unlimited" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    echo "* - nofile $(cat /proc/sys/fs/nr_open)" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    echo "* - nproc 96118" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    echo "* - as unlimited" | sudo tee -a /etc/security/limits.d/10-audius.conf
+    sudo sysctl -p
```

Comment:
i think you may have to add the `/etc/security/limits.d/10-audius.conf` file here bc it reads from /etc/sysctl.conf as a default?

Reply:
https://unix.stackexchange.com/a/379341 in combination with GCP enabling pam_limits.so by default allows us to use the `/etc/security/limits.d` path. The added benefit is having all the audius-specific settings in one file.